code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1408,20 +1408,26 @@
 
 /datum/reagent/medicine/sanguiose
 	name = "Sanguiose"
-	description = "A chemical developed to aid in the butchering proccess, it causes a chemical reaction which consumes blood and oxygen while healing cuts, bruises, and other similar injuries,"
+	description = "A chemical developed to aid in the butchering proccess, it causes a chemical reaction which heals bruises and other similar injuries. Has a moderate chance of causing muteness in the user."
 	reagent_state = LIQUID
 	color = "#FF6464"
 	metabolization_rate = 0.5* REAGENTS_METABOLISM
 	overdose_threshold = 25
 	taste_description = "salty"
+	
 
 /datum/reagent/medicine/sanguiose/on_mob_life(mob/living/carbon/M)
 	M.adjustBruteLoss(-1, 0)
-	M.adjustOxyLoss(0.25,0)
-	M.blood_volume -= 1 //Removes blood
+	if(prob(1))
+		ADD_TRAIT(M, TRAIT_MUTE, MEDICINE_TRAIT)
```

Comment:
prob want to source it to sanguiose instead of generic medicine so it won't interfere with others.

Reply:
really this is not a good cat 2 balancer since the prob is either too low or it makes for a good mute chem.